client/src/components/Week/Week.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import '../../../node_modules/bulma/css/bulma.css';
+import moment from 'moment';
+import styled from 'styled-components';
+import './Week.css'
+
+import { Day } from '../';
+
+const Distance = styled.div`
+  font-size: 2em;
+  font-weight: bold;
+  text-align: right;
+`
+const Unit = styled.div`
+  display: inline;
+  font-size: .5em;
+  font-weight: normal;
+`
+const Percentage = styled.div`
+  font-size: 2em;
+  font-style: italic;
+  text-align: right;
+`
+
+function Week({ runData, unit, DateBox, H3, weekDayNums, FirstBoxKpi }) {
+
+
+  const maxThreshold = 30;
+  const longestPrct = Math.round(runData.longestRun  / runData.total * 100);
+
+  function filterRunsByDate(runArr, date) {
+    return runArr.filter(run => moment(new Date(run.date)).isSame(date, 'day'));
+  }
+
+  return (
+    <div className="columns is-mobile">
+      {
+        weekDayNums.map(day => {
+          const calendarDate = moment(new Date(runData.week)).add(day, 'days');
+          const runArr = filterRunsByDate(runData.runs, calendarDate);
+          return (
+            <Day key={calendarDate} DateBox={DateBox} H3={H3} Distance={Distance} Unit={unit} runArr={runArr} 
```

Comment:
It's interesting how you pass the components down to be able to style them, that's some kind of inversion of control. For now, the community has been solving this by using an advanced React pattern called [Compound Components](https://medium.com/@Dane_s/react-js-compound-components-a6e54b5c9992)